__tests__/app.test.js

```diff
@@ -105,3 +105,20 @@ describe('PATCH /api/articles/:article_id', () => {
     });
   });
 });
+
+describe('GET /api/users', ()=>{
+  test(`Status 200: Responds with an array of objects, each object should have the following property:
+username`, () => {
+    return request(app).get('/api/users').expect(200).then(({body})=>{
+      expect(body.users).toBeInstanceOf(Array);
+      body.users.forEach((user)=>{
+        expect(user).toHaveProperty('username');
+      });
+    });
+  });
+  test(`Status 404: Mis-spelled url`, () => {
+    return request(app).get('/api/handbag').expect(404).then(({body})=>{
+      expect(body).toHaveProperty('message', 'Not Found');
+    });
```

Comment:
Is this test specific to this endpoint?

Reply:
Probably not neccessary at all in fact. My 404 error in my app will trigger the same response